doc/configdb-schema.md

```diff
@@ -0,0 +1,226 @@
+
+    Example:
+    127.0.0.1:6379[4]> keys BGP_NEIGHBOR:*
+     1) "BGP_NEIGHBOR:10.0.0.31"
+     2) "BGP_NEIGHBOR:10.0.0.39"
+     3) "BGP_NEIGHBOR:10.0.0.11"
+     4) "BGP_NEIGHBOR:10.0.0.7"
+     5) "BGP_NEIGHBOR:10.0.0.15"
+     6) "BGP_NEIGHBOR:10.0.0.45"
+     7) "BGP_NEIGHBOR:10.0.0.35"
+     8) "BGP_NEIGHBOR:10.0.0.51"
+     9) "BGP_NEIGHBOR:10.0.0.1"
+    10) "BGP_NEIGHBOR:10.0.0.43"
+    11) "BGP_NEIGHBOR:10.0.0.3"
+    12) "BGP_NEIGHBOR:10.0.0.21"
+    13) "BGP_NEIGHBOR:10.0.0.55"
+    14) "BGP_NEIGHBOR:10.0.0.17"
+    15) "BGP_NEIGHBOR:10.0.0.19"
+    16) "BGP_NEIGHBOR:10.0.0.25"
+    17) "BGP_NEIGHBOR:10.0.0.63"
+    18) "BGP_NEIGHBOR:10.0.0.41"
+    19) "BGP_NEIGHBOR:10.0.0.29"
+    20) "BGP_NEIGHBOR:10.0.0.33"
+    21) "BGP_NEIGHBOR:10.0.0.5"
+    22) "BGP_NEIGHBOR:10.0.0.23"
+    23) "BGP_NEIGHBOR:10.0.0.13"
+    24) "BGP_NEIGHBOR:10.0.0.27"
+    25) "BGP_NEIGHBOR:10.0.0.57"
+    26) "BGP_NEIGHBOR:10.0.0.47"
+    27) "BGP_NEIGHBOR:10.0.0.37"
+    28) "BGP_NEIGHBOR:10.0.0.53"
+    29) "BGP_NEIGHBOR:10.0.0.49"
+    30) "BGP_NEIGHBOR:10.0.0.9"
+    31) "BGP_NEIGHBOR:10.0.0.61"
+    32) "BGP_NEIGHBOR:10.0.0.59"
+    127.0.0.1:6379[4]> hgetall BGP_NEIGHBOR:10.0.0.13
+    1) "admin_status"
+    2) "up"
+    3) "peer_addr"
+    4) "10.0.0.12"
+    5) "asn"
+    6) "65200"
+    7) "name"
+    8) "ARISTA07T2"
+
+### BGP_PEER_RANGE table
+
+    ; Key
+    bgp_peer_range_key    = 1*64VCHAR
+    ; Attributes
+    name                  = 1*64VCHAR
+    ip_range              = LIST(IPPrefix)
+
+
+### MGMT_INTERFACE table
+
+    ; Key
+    port_name             = 1*64VCHAR            ; name of mgmt port that the ip interface attach to
+    ip_prefix             = IPPrefix    
+    ; Attributes
+    gwaddr                = IPAddress  
+    forced_mgmt_routes    = LIST(IPPrefix)       ; the prefixes to which route will be force to be going through mgmt port
+    
+
+### LOOPBACK_INTERFACE table
+
+    ; Key
+    port_name              = 1*64VCHAR
+    ip_prefix              = IPPrefix
+    ; Attributes
+    ; No attributes
+    
+
+### PORT table
+
+    ; Key
+    name                  = 1*64VCHAR
+    ; Attributes
+    alias                 = 1*64VCHAR
+    MTU                   = 1*5DIGIT  
+    front_panel_index     = 1*3DIGIT
+    speed                 = 1*5DIGIT  
+    
+### INTERFACE table
+
+    ; Key
+    port_name              = 1*64VCHAR
+    ip_prefix              = IPPrefix
+    ; Attributes
+    ; No attributes
+    
+### PORTCHANNEL table
+
+    ; Key
+    name                  = 1*64VCHAR
+    ; Attributes
+    members               = LIST(1*64VCHAR)
+    
+### PORTCHANNEL_INTERFACE table
+
+    ; Key
+    portchannel_name      = 1*64VCHAR
+    ip_prefix             = IPPrefix
+    ; Attributes
+    ; No attributes
+
+### VLAN table
+
+    ; Key
+    name                 = 1*64VCHAR
+    ; Attributes
+    id                   = 1*5DIGIT 
+    members              = LIST(1*64VCHAR)
+    
+### VLAN_INTERFACE table
```

Comment:
why do we have different interface table for VLAN? why not inside the INTERFACE table?